package com.example.loot.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@lombok.NoArgsConstructor
@Data
@AllArgsConstructor
public class CookingDTO {

    private String name;

    private String description;

    private String instructions;

    private String category;

    private List<String> ingredients;

    private List<MissingIngredientDTO> missing;
}